sdk/e2e/12_deploy_router_upi_std_ensembler_test.py

```diff
@@ -126,8 +126,9 @@ def test_deploy_router_upi_std_ensembler():
     channel = grpc.insecure_channel(retrieved_router.endpoint)
     stub = upi_pb2_grpc.UniversalPredictionServiceStub(channel)
 
-    logging.info("send request that satisfy treatment-a")
+    logging.info("send request that satisfy control")
```

Comment:
other than the traffic rules splt which has some flaw and the others are merely adapting to the new corrected config